common/src/main/java/tachyon/worker/WorkerNetAddress.java

```diff
@@ -40,6 +41,18 @@ public WorkerNetAddress(String host, int rpcPort, int dataPort, int webPort) {
   }
 
   /**
+   * Constructs the worker net address from thrift construct.
+   *
+   * @param netAddress the thrift net address
+   */
+  public WorkerNetAddress(NetAddress netAddress) {
```

Comment:
Rename the thrift struct to WorkerNetAddress sounds good to me.

Why do you prefer WorkerNetConf? I think it's common that an address consists of hostname and ports?